prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -2,53 +2,91 @@
 
 layout: page_v2
 title: Custom or No mediation
-description: Integration of Prebid SDK withou primaty Ad Server
+description: Android SDK Custom Bidding Integration
 sidebarType: 2
 
 ---
 
-# Custom Bidding Integration
+
+
+# Android SDK Custom Bidding Integration
 {:.no_toc}
 
-You can use Prebid SDK to monetize your app with a custom ad server or even without it. Use the `Transport API` to obtain the targeting keywords for following usage with the custom ad server. Use the `Rendering API` to display the winning bid without primary ad server and its SDK.
+## Introduction
 
 * TOC
+
 {:toc}
 
+In this article, we will show you how to do the following:
+
+1. Initialize the Prebid Mobile Android SDK.
+2. Set up various types of AdUnits.
+3. How to initialize the auction.
+4. Finally, how to render the results.
+
+## Initialization
+
+Before implementing the instructions in this guide, you need to ensure that you have correctly initialized the Prebid Mobile SDK in your app. The initialization method has the following pattern:
+
+```java
+Prebid.init(context, accountID, adServer, host);
+```
+
+To use the above code snippet, you will need to [setup a Prebid server.](https://docs.prebid.org/prebid-mobile/prebid-mobile-getting-started.html#set-up-prebid-server) You can either register with a [Prebid.org member that hosts Prebid Server](https://prebid.org/managed-services/) or [setup your own Prebid server.](https://docs.prebid.org/prebid-server/hosting/pbs-hosting.html)
+
+* `context`: This parameter represents the Context of the Android application. It is typically obtained using `getApplicationContext()` or `this` (if called from within an Activity). The Context is essential for the SDK to interact with the Android environment.
+
+* `adUnitId`: This is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".
+
+* `accountId`: The `accountId` you use to initialize the Prebid Mobile SDK is the unique identifier your Prebid Server provider assigned to you. This links requests from your app to server-side settings such as timeout, price granularity, and others.
```

Comment:
Please add: "Note: the accountId is also used to define the "account settings" used for this app in Prebid Server, which defines attributes like timeout and price granularity."

Please update this in both android and ios. We'll soon be able to enhance this, but we can start simple for now.